kilkari/src/main/java/org/motechproject/nms/kilkari/domain/WeeklyCallsNotAnsweredMsisdnRecord.java

```diff
@@ -0,0 +1,51 @@
+package org.motechproject.nms.kilkari.domain;
+
+import org.joda.time.DateTime;
+import org.motechproject.mds.annotations.Entity;
+import org.motechproject.mds.annotations.Field;
+
+import javax.jdo.annotations.Column;
+import javax.jdo.annotations.Unique;
+import javax.validation.constraints.Max;
+import javax.validation.constraints.Min;
+
+/**
+ * Record of Weekly Calls Not Answered Deactivated Numbers
+ */
+@Entity(tableName = "nms_weekly_calls_not_answered_msisdn")
```

Comment:
I'm a little concerned with the naming here. This table could contain a lot of msisdn that were deativated for reasons other not answered calls. New requests could come in for other reasons. 

Ideally, this would be called a blacklist table or some other name and then add another column with the last deactivation reason.